clarity/contracts/arkadiko-stacker-payer-v3-4.clar

```diff
@@ -0,0 +1,137 @@
+;; Implementation of Stacker Payer
+;; which allows users to redeem xSTX for STX
+
+(define-constant ERR-NOT-AUTHORIZED u22401)
+(define-constant ERR-EMERGENCY-SHUTDOWN-ACTIVATED u221)
+(define-constant ERR-VAULT-ALREADY-REDEEMED u222)
+(define-constant ERR-AUCTION-RUNNING u223)
+(define-constant ERR-WRONG-COLLATERAL u224)
+(define-constant ERR-NOT-LIQUIDATED u225)
+
+(define-data-var stacker-payer-shutdown-activated bool false)
+(define-data-var stx-redeemable uint u45650000000) ;; TODO: update for mainnet
```

Comment:
Q: I forgot why we have this contract. Freddie is also keeping stx-redeemable and has a methods "redeem-stx", "release-stacked-stx"

Reply:
@philipdesmedt In any case I think we can simplify this. The latest liquidation update makes sure a user can only claim xSTX once the liquidated vault has stopped stacking. So we can simplify and always allow swapping xSTX for STX (less admin work for us). Later we can optimise this so the user does not need to claim xSTX and swap himself, but it is just done in 1 tx.

Reply:
Have simplified the stacker payer.